src/AlgebraicGeometry/Schemes/SpecOpen/Objects/Attributes.jl

```diff
@@ -105,7 +104,10 @@ of ``U``. This function can also be called using the
 `getindex` method or simply via `U[i]`.
 """
 affine_patch(U::SpecOpen, i::Int) = affine_patches(U)[i]
+gens(U::SpecOpen) = affine_patches(U)
+gen(U::SpecOpen, i::Int) = affine_patches(U)[i]
 getindex(U::SpecOpen, i::Int) = affine_patches(U)[i]
+ngens(U::SpecOpen) = length(U.gens)
```

Comment:
So is `length(U.gens)` always equal to `length(affine_patches(U))` ?

Looking above, I see that `npatches` is defined to also return `length(U.gens)`, so I'll assume this answers my question in the affirmative. But if so, I think it would be cleaner to do this:
```suggestion
ngens(U::SpecOpen) = npatches(U)
```